setup.py

```diff
@@ -77,7 +77,6 @@
     "invenio-db[postgresql]>=1.0.5,<1.1.0",
     "SQLAlchemy-Utils[encrypted]>=0.33.0,<0.36.0",
     "six>=1.12.0",  # required by Flask-Breadcrumbs
-    "celery<5.0.0",
```

Comment:
With this PR and [reana-commons'](https://github.com/reanahub/reana-commons/pull/332) checked out I still get `celery==4.4.7` installed.

```console
$ pip show celery      
Name: celery
Version: 4.4.7
Summary: Distributed Task Queue.
Home-page: http://celeryproject.org
Author: Ask Solem
Author-email: auvipy@gmail.com
License: BSD
Location: /Users/marco/.virtualenvs/r-server/lib/python3.8/site-packages
Requires: billiard, kombu, pytz, vine
Required-by: Flask-CeleryExt, invenio-celery
```

Can you reproduce it?

Reply:
I managed to install kombu and celery 5.x by commenting out r-commons and r-db, and installing it via DEBUG. I did it to test if the platform in general works. But to make it "nice" we need to release r-commons without the pin on the kombu version.

Reply:
Try replace `../reana-commons` with `".[all]" --upgrade`.

Reply:
There is no `>= 5.x` requirement in any invenio packages. If we want to have celery of 5.x for sure, we need to pin it.

Reply:
Sorry, I'm not sure I understand 😕. I'm doing that in the previous step. By installing `../reana-commons` I'm bringing the changes introduced in your PR to my local environment.

Reply:
I tried to uninstall all pip packages from virtual env:

```bash
$ pip freeze > to_delete.txt
$ pip uninstall -r to_delete.txt -y
```

And then followed the steps above:

```bash
$ cd reana-server
$ pip install ../reana-commons --no-cache-dir  # with PR, this should install latest kombu first
$ pip install ".[all]" --upgrade  # this will install the rest of the packages from r-server and use latest kombu version
$ pip show celery
```

Got this:

```bash
$ pip show celery
Name: celery
Version: 5.1.2
Summary: Distributed Task Queue.
...
```

Reply:
But.. `5.1.2` is vulnerable 🙁
https://github.com/reanahub/reana-server/security/dependabot/requirements.txt/celery/open

Reply:
Then after bumping r-commons, we will need to specify the celery `5.2.x` version in `requirements.txt` I suppose.

Reply:
Looks like after the changes it installs a non-vulnerable version 👍 

```console
$ pip show celery
Name: celery
Version: 5.2.3
```